packages/application-shell/src/components/application-shell/application-shell.spec.js

```diff
@@ -846,7 +846,7 @@ describe('when clicking on navbar menu toggle', () => {
     });
 
     // Check that the "Support" link is rendered when the menu is expanded
-    expect(navbarRendered.getByText('Support')).toBeInTheDocument();
+    expect(navbarRendered.getAllByText('Support')).toHaveLength(1);
```

Comment:
Actually here, because we are checking when expanded, we have only 1 support element in the dom from the Support label and not from the tooltip.
`{!isMenuOpen && <SupportMenuTooltipContainer>...`

Reply:
I pushed a commit in order to improve tests assertions: 5fe19b4977bce8c436e1555de4032ec0a04731c4

Reply:
Thank you 🙏🏾